window_Popup.js

```diff
@@ -0,0 +1,423 @@
+    workDetails: ['FACEBOOK', 'Back End Dev', '2015'],
+    image: './images/Portfolio 4.png',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    details:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    details2:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+
+    workTools: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+
+    liveIcon: './images/live-Icon.svg',
+    liveLink: 'https://felixdev22.github.io/Mobile-first-D3/',
+
+    sourceLink: 'https://github.com/felixDev22/Mobile-first-D3',
+    sourceIcon: './images/bluegithub.svg',
+  },
+];
+
+// ------------------------- Projects section --------------------//
+
+const project = document.querySelector('.grid-container');
+project.innerHTML = `<div class="card">
+              <div class="row">
+                <div class="col-1 work-1">
+                <a href="#"
+                  ><img src="${portfolioData[1].image}" class="mobile-only" alt="portfolio-1"
+                /></a>
+  
+                   <a href="#"
+                  ><img src="${portfolioData[1].image2}" class="desktop-only" alt="portfolio-1"
+                /></a>
+                
+                </div>
+
+                <div class="col-2">
+                <a href="#"><h2>${portfolioData[0].title}</h2></a>
+                <div class="work-details flex2">
+                  <a href="#"><h3>${portfolioData[0].workDetails[0]}</h3></a>
+                  <div class="bullet-point"></div>
+                  <a href=""><h4>${portfolioData[0].workDetails[1]}</h4></a>
+                  <div class="bullet-point"></div>
+                  <a href=""><h4>${portfolioData[0].workDetails[2]}</h4></a>
+                </div>
+                <p>
```

Comment:
While you have done a good job in implementing the cards and the pop up with JavaScript, you are still hand coding the data in a static way instead of dynamic. You'd realize that if you had 20 more cards to create, you'd be writing these semantics 20 times. I suggest you take advantage of the JavaScript loops here since they help you to do things repetitively. This will help your code to become dynamic and less repetitive.


NOTE: this suggestion applies the the way the modal is being created.